### ISSUE TITLE
JS TypeError on mobile

### ISSUE THREAD

**Issue by jeffsta9** [codersaiful/woo-product-table#277]
WP v 6.1.1
WPT v3.3.5

The error:
Uncaught TypeError: Cannot read properties of undefined (reading 'data')
- at HTMLBodyElement.<anonymous> (woo-custom-features.min.js?ver=3.4.1:1:22100)
- at HTMLBodyElement.dispatch (jquery.min.js?ver=3.6.1:2:43064)
- at y.handle (jquery.min.js?ver=3.6.1:2:41048)
- at Object.trigger (jquery.min.js?ver=3.6.1:2:71659)
- at HTMLBodyElement.<anonymous> (jquery.min.js?ver=3.6.1:2:72254)
- at Function.each (jquery.min.js?ver=3.6.1:2:3003)
- at s.fn.init.each (jquery.min.js?ver=3.6.1:2:1481)
- at s.fn.init.trigger (jquery.min.js?ver=3.6.1:2:72230)
- at HTMLDocument.<anonymous> (custom.js?ver=3.3.5.3:380:60)
- at e (jquery.min.js?ver=3.6.1:2:30038)

Works fine on Win browsers but not on Mac or mobile FF/Chrome. 

- Select all is not working
- Add selected to cart is not working

Any suggestions?

Thanks in advance! :)

**Comment by fazlebarisn** [comment]
Please share your page link.

**Comment by jeffsta9** [comment]
https://plumbinglabels.com/individual-labels/

**Comment by fazlebarisn** [comment]
I am sorry but I am not getting the error. 
Can you please give me a screenshot? Which file and which line you commented on?
Or just recode a video.
Thanks

**Comment by clubztutoring** [comment]
Yeah, had to comment out the troublesome lines to remove the error as it was causing the Check All and Add Selected to Cart buttons to not function. I tracked it down via the trace in my initial message above to [woo-product-table/assets/js/custom.js](https://plumbinglabels.com/wp-content/plugins/woo-product-table/assets/js/custom.js) in the following line...

380    $( document.body ).trigger( 'removed_from_cart' );

I just commented out the entire block

```
//wc_fragments_refreshed,wc_fragments_refresh,wc_fragment_refresh,removed_from_cart
// $( document.body ).trigger( 'updated_cart_totals' );
// $( document.body ).trigger( 'wc_fragments_refreshed' );
// $( document.body ).trigger( 'wc_fragments_refresh' );
// $( document.body ).trigger( 'wc_fragment_refresh' );
_**// $( document.body ).trigger( 'removed_from_cart' );**_
```

After the cache finally cleared, everything works as expected. 

This is for a friend's site - I'm just helping him out.

**Comment by jeffsta9** [comment]
Yeah, had to comment out the troublesome lines to remove the error as it was causing the Check All and Add Selected to Cart buttons to not function. I tracked it down via the trace in my initial message above to [woo-product-table/assets/js/custom.js](https://plumbinglabels.com/wp-content/plugins/woo-product-table/assets/js/custom.js) in the following line...

`379    $( document.body ).trigger( 'removed_from_cart' );`

I just commented out the entire block

```
//wc_fragments_refreshed,wc_fragments_refresh,wc_fragment_refresh,removed_from_cart
// $( document.body ).trigger( 'updated_cart_totals' );
// $( document.body ).trigger( 'wc_fragments_refreshed' );
// $( document.body ).trigger( 'wc_fragments_refresh' );
// $( document.body ).trigger( 'wc_fragment_refresh' );
// $( document.body ).trigger( 'removed_from_cart' );
```

After the cache finally cleared, everything works as expected. 

This is for a friend's site - I'm just helping him out.
